.clusterfuzzlite/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -eu
+# Supply build instructions
+# Use the following environment variables to build the code
+# $CXX:               c++ compiler
+# $CC:                c compiler
+# CFLAGS:             compiler flags for C files
+# CXXFLAGS:           compiler flags for CPP files
+# LIB_FUZZING_ENGINE: linker flag for fuzzing harnesses
+
+# Copy all fuzzer executables to $OUT/
+
+# Copy all fuzzer executables to $OUT/
+$CXX $CFLAGS $LIB_FUZZING_ENGINE \
```

Comment:
I only see `$CFLAGS`... should there also be `$CXXFLAGS`

Reply:
Ah, good catch. It should be `CXXFLAGS` and not `CFLAGS` used for building.